app/views/miq_request/_pre_prov.html.haml

```diff
@@ -1,5 +1,6 @@
 #pre_prov_div
   - typ = request.parameters[:controller] == "vm_cloud" ? ui_lookup(:table => "template_cloud") : ui_lookup(:table => "template_infra")
+  = react 'CustomSearchBar', onSearch: -> (searchQuery) { handleSearch(searchQuery) }
```

Comment:
where is `handleSearch` defined?

Reply:
I guess you are sending the data from `handleSearch` to the `def pre_prov`. but I am not sure how it's being connected.